ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml

```diff
@@ -32,7 +32,9 @@ workflow:
     pre:
     - ref: hypershift-kubevirt-baremetalds-conf-devscripts
     - chain: baremetalds-ipi-pre
-    - ref: hypershift-mce-install-disconnected
```

Comment:
cannot we drop all the files for this step now?

Reply:
It will be deleted because the agent has not been fully adjusted yet. Once the agent is fully adjusted, I will delete this step.